packages/example-app-base/src/components/App/App.tsx

```diff
@@ -20,13 +21,23 @@ const getClassNames = classNamesFunction<IAppStyleProps, IAppStyles>();
 export class AppBase extends React.Component<IAppProps, IAppState> {
   public state: IAppState = { isMenuVisible: false };
   private _classNames: IProcessedStyleSet<IAppStyles>;
+  private _baseUrl: string;
+
+  constructor(props: IAppProps) {
+    super(props);
+
+    const doc = getDocument();
+    this._baseUrl = doc ? document.location.href : '';
+  }
 
   public render(): JSX.Element {
     const { appDefinition, styles, responsiveMode = ResponsiveMode.xLarge, theme } = this.props;
     const { customizations } = appDefinition;
     const { isMenuVisible } = this.state;
 
-    const classNames = (this._classNames = getClassNames(styles, { responsiveMode, theme }));
+    const showOnlyExamples = this._baseUrl.indexOf('docsExample=true') > -1;
```

Comment:
Can you give an example of what a full URL with this query in it would look like?

Reply:
(mainly meant in a comment, but in code too wouldn't hurt)

Reply:
[Here is a link](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/9438/merge/fabric-website-resources/dist/index.html#/examples/button?docsExample=true) to a dev deploy site out of this PR. The url string will be eventually: *http:/fabricweb.z5.web.core.windows.net/pr-deploy-site/**refs/heads/master**/fabric-website-resources/dist/index.html#/examples/button?docsExample=true* so it does not depend on a pull request. I actually see it as something to be used by myself when wanting to work on a component and see nothing else around.
[Here is link](https://review.docs.microsoft.com/en-us/javascript/docs/examples/button?view=office-ui-fabric-react-latest&branch=testPackageSetup) to the docs page where we use this url as a source for an iframe.